src/core/contrib.js

```diff
@@ -23,8 +23,18 @@ function findUserURLs(...thingsWithUsers) {
   return [...new Set(usersURLs)];
 }
 
+function getHeaders(conf) {
```

Comment:
This extracts the GitHub credential,s and returns them as a header... so `extractGithubCredentials(conf)`

Reply:
This function seems like it should live in the Github API module... not in contrib.